packages/test/test-end-to-end-tests/src/test/summaries.spec.ts

```diff
@@ -326,67 +323,57 @@ describeNoCompat("Summaries", (getTestObjectProvider) => {
 		);
 	});
 
-	itExpects(
-		"full initialization of data object should not happen by default",
-		[
-			{
-				eventName: "fluid:telemetry:Container:Request_cancel",
-				error: "Non interactive/summarizer client's data object should not be initialized",
-			},
-		],
-		async () => {
-			const dataStoreFactory1 = new DataObjectFactory(
-				"@fluid-example/test-dataStore1",
-				TestDataObject1,
-				[],
-				[],
-			);
-			const registryStoreEntries = new Map<string, Promise<IFluidDataStoreFactory>>([
-				[dataStoreFactory1.type, Promise.resolve(dataStoreFactory1)],
-			]);
-			const runtimeFactory = new ContainerRuntimeFactoryWithDefaultDataStore({
-				defaultFactory: dataStoreFactory1,
-				registryEntries: registryStoreEntries,
-			});
-
-			// Create a container for the first client.
-			const container1 = await provider.createContainer(runtimeFactory);
-			await assert.doesNotReject(
-				requestFluidObject<TestDataObject1>(container1, "/"),
-				"Initial creation of container and data store should succeed.",
-			);
+	it("full initialization of data object should not happen by default", async () => {
+		const dataStoreFactory1 = new DataObjectFactory(
+			"@fluid-example/test-dataStore1",
+			TestDataObject1,
+			[],
+			[],
+		);
+		const registryStoreEntries = new Map<string, Promise<IFluidDataStoreFactory>>([
+			[dataStoreFactory1.type, Promise.resolve(dataStoreFactory1)],
+		]);
+		const runtimeFactory = new ContainerRuntimeFactoryWithDefaultDataStore({
+			defaultFactory: dataStoreFactory1,
+			registryEntries: registryStoreEntries,
+		});
 
-			// Create a summarizer for the container and do a summary shouldn't throw.
-			const createSummarizerResult = await createSummarizerFromFactory(
-				provider,
-				container1,
-				dataStoreFactory1,
-				undefined,
-				ContainerRuntimeFactoryWithDefaultDataStore,
-				registryStoreEntries,
-			);
-			await assert.doesNotReject(
-				summarizeNow(createSummarizerResult.summarizer, "test"),
-				"Summarizing should not throw",
-			);
+		// Create a container for the first client.
+		const container1 = await provider.createContainer(runtimeFactory);
+		await container1.getEntryPoint();
```

Comment:
Should't this be wrapped in an `assert.doesNotReject` to keep the behavior same as before?

Reply:
Not sure what happened before. When I first fixed these tests, it didn't like the `rejects/doesNotReject` calls.